implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -143,6 +146,21 @@ default boolean useDynatraceSummaryInstruments() {
         return getBoolean(this, "useDynatraceSummaryInstruments").orElse(true);
     }
 
+    /**
+     * Toggle whether to export metadata (unit and description) to the Dynatrace backend
+     * for the V2 version of this exporter. Metadata will be exported by default from
+     * Micrometer version 1.12.0. This setting has no effect for the (legacy) Dynatrace
+     * Exporter v1
+     * @return true if metadata should be exported, false otherwise.
+     * @since 1.12.0
+     */
+    default boolean exportMetadata() {
```

Comment:
Hum I'm not but `exportMetadata` and `enrichWithDynatraceMetadata` are more or less similar? Thinking if users won't be confused by this new setting now. Do we plan to export more metadata than unit and descriptions? Maybe `exportUnitDescription` is clearer? (although not a great name..). Or `exportMeterMetadata`?

Reply:
Thats a good point. Currently, Micrometer does not provide any metadata apart from unit and description, but I would like to stay with a more general name. I do like `exportMeterMetadata`.